docs/UserGuide.md

```diff
@@ -39,7 +39,7 @@ This project is built on top of [AddressBook-Level3 Project](https://se-educatio
    Note: A GUI similar to the image below should appear in a few seconds. The image below consist of existing sample data.<br>
    </box>
 
-   ![defaultUi](/images/uiWithDescription.png)
+   ![defaultUi](images/uiWithDescription.png)
```

Comment:
Good job spotting and fixing the bug on the image path!

Reply:
Nice thanks for the change!